src/modules/projects/components/tables/ProjectClientEnrollmentsTable.tsx

```diff
@@ -178,20 +189,33 @@ const ProjectClientEnrollmentsTable = ({
     [openOnDate]
   );
 
+  const {
+    project: { staffAssignmentsEnabled },
+  } = useProjectDashboardContext();
+
   const defaultColumns: ColumnDef<ProjectEnrollmentQueryEnrollmentFieldsFragment>[] =
     useMemo(() => {
-      return [
+      const c = [
```

Comment:
I know this is a small function but suggest we use more descriptive variable names, such as `cols` or just `results`